static/redfish/v1/schema/OemAssembly_v1.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<edmx:Edmx xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx" Version="4.0">
+    <edmx:Reference Uri="http://docs.oasis-open.org/odata/odata/v4.0/errata03/csd01/complete/vocabularies/Org.OData.Core.V1.xml">
+        <edmx:Include Namespace="Org.OData.Core.V1" Alias="OData"/>
+    </edmx:Reference>
+    <edmx:Reference Uri="http://redfish.dmtf.org/schemas/v1/Assembly_v1.xml">
+        <edmx:Include Namespace="Assembly"/>
+        <edmx:Include Namespace="Assembly.v1_0_0"/>
+    </edmx:Reference>
+    <edmx:Reference Uri="http://redfish.dmtf.org/schemas/v1/RedfishExtensions_v1.xml">
+        <edmx:Include Namespace="RedfishExtensions.v1_0_0" Alias="Redfish"/>
+    </edmx:Reference>
+
+    <edmx:DataServices>
+
+        <Schema xmlns="http://docs.oasis-open.org/odata/ns/edm" Namespace="OemAssembly">
+            <Annotation Term="Redfish.OwningEntity" String="IBM"/>
+        </Schema>
+
+        <Schema xmlns="http://docs.oasis-open.org/odata/ns/edm" Namespace="OemAssembly.v1_0_0">
+            <Action Name="ReadyToRemove" IsBound="true">
+                <Annotation Term="OData.Description" String="An indication of whether the system is prepared for the assembly to be changed."/>
+                <Annotation Term="OData.LongDescription" String="This property shall indicate whether the system is ready for the assmebly to be changed.  Setting the value to `true` shall cause the service to perform appropriate actions to stop monitoring the battery voltage.  Setting the value to `false` shall cause the service to perform appropriate actions to resume monitoring the battery voltage."/>
+                <Parameter Name="OemAssembly" Type="OemAssembly.v1_0_0.OemActions"/>
```

Comment:
Please don't use anything specific to battery voltage so this can also be used for op panel